web-src/src/common/components/history/executions-log-table.vue

```diff
@@ -3,11 +3,21 @@
     <table class="highlight striped">
       <thead>
       <tr>
-        <th class="id-column">ID</th>
-        <th class="start_time-column">Start Time</th>
-        <th class="user-column">User</th>
-        <th class="script-column">Script</th>
-        <th class="status-column">Status</th>
+        <th class="id-column">
+          <a href="#" @click="sortBy('id')">ID<span :class="ascending ? 'arrow asc' : 'arrow desc'"></span></a>
```

Comment:
Instead of `arrow asc` I would suggest using just `asc`. The same for `desc`.

Reply:
How should the `css` part look like if I drop `arrow`?

Reply:
Instead of `.arrow.asc` you could use `th.asc` (tag name)

Reply:
Ah, I got what you mean. Probably you could just rename "arrow" to "sorted" ?

Reply:
Sure thing, done.